### PR TITLE
fix: Remove conda install instrictions from docs

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

**Review by jacobtaxen — COMMENTED** [review]
I think it would be better to mention Conda, i.e. that you can install it with a pip within the conda environment

**Comment by abhilash-kumar-nair** [comment]
@efredriksson-modelon , Could you take please take a sneak peak into this minor addition

File: docs/source/install.rst [hunk 1]
```diff
@@ -10,6 +10,14 @@ You can install the client with ``pip``::
 
    pip install modelon-impact-client
 
+Conda
+-----
+
+The modelon-impact-client package is not hosted on any conda channels. However, you can still install the client 
+with ``conda`` using :ref:`pip <Pip>`. Refer
```
Comment (efredriksson-modelon) [inline docs/source/install.rst:17]: Refer -> Refer to?

File: docs/source/install.rst [hunk 2]
```diff
@@ -10,6 +10,14 @@ You can install the client with ``pip``::
 
    pip install modelon-impact-client
 
+Conda
+-----
+
+The modelon-impact-client package is not hosted on any conda channels. However, you can still install the client 
+with ``conda`` using :ref:`pip <Pip>`. Refer 
+(`using pip in an environment <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#using-pip-in-an-environment>`_)
```
Comment (efredriksson-modelon) [inline docs/source/install.rst:18]: Would it not be nicer to remove the parentheses around the link?

**Review by efredriksson-modelon — APPROVED** [review]
That is very good! 🧇 Left some style suggestions.

**Pull request merged**